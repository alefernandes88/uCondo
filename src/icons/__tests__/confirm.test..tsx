import React from 'react'
import { render } from '@testing-library/react-native'
import ConfirmIcon from '../confirm'

describe('Icons -> Back', () => {
  it('should render with a normal style', () => {
    const { toJSON } = render((
      <ConfirmIcon />
    ))

    expect(toJSON()).toMatchSnapshot()
  })

  it('should render with a custom style', () => {
    const { toJSON } = render((
      <ConfirmIcon
        color="red"
        height={30}
        width={30}
        strokeSize={1.5}
      />
    ))

    expect(toJSON()).toMatchSnapshot()
  })
})
