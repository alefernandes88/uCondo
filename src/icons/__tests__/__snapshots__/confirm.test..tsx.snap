// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icons -> Back should render with a custom style 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="30"
  bbWidth="30"
  fill="none"
  focusable={false}
  height={30}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 30,
        "width": 30,
      },
    ]
  }
  vbHeight={16}
  vbWidth={20}
  width={30}
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGPath
      d="M19.375 0.453125H17.7367C17.507 0.453125 17.2891 0.558594 17.1484 0.739062L7.48517 12.9805L2.85157 7.10938C2.78148 7.02038 2.69214 6.94842 2.59025 6.8989C2.48836 6.84937 2.37658 6.82357 2.26329 6.82344H0.625013C0.467982 6.82344 0.381263 7.00391 0.477357 7.12578L6.89689 15.2586C7.19689 15.6383 7.77345 15.6383 8.07579 15.2586L19.5227 0.753125C19.6188 0.633594 19.532 0.453125 19.375 0.453125Z"
      fill={
        {
          "payload": 4294901760,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;

exports[`Icons -> Back should render with a normal style 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="16"
  bbWidth="20"
  fill="none"
  focusable={false}
  height={16}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 16,
        "width": 20,
      },
    ]
  }
  vbHeight={16}
  vbWidth={20}
  width={20}
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGPath
      d="M19.375 0.453125H17.7367C17.507 0.453125 17.2891 0.558594 17.1484 0.739062L7.48517 12.9805L2.85157 7.10938C2.78148 7.02038 2.69214 6.94842 2.59025 6.8989C2.48836 6.84937 2.37658 6.82357 2.26329 6.82344H0.625013C0.467982 6.82344 0.381263 7.00391 0.477357 7.12578L6.89689 15.2586C7.19689 15.6383 7.77345 15.6383 8.07579 15.2586L19.5227 0.753125C19.6188 0.633594 19.532 0.453125 19.375 0.453125Z"
      fill={
        {
          "payload": 4294967295,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
